api/envoy/extensions/filters/http/api_key_auth/v3/api_key_auth.proto

```diff
@@ -17,38 +16,95 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 option (xds.annotations.v3.file_status).work_in_progress = true;
 
 // [#protodoc-title: APIKey Auth]
-// [#not-implemented-hide:]
 // APIKey Auth :ref:`configuration overview <config_http_filters_api_key_auth>`.
 // [#extension: envoy.filters.http.api_key_auth]
 
 // API Key HTTP authentication.
 //
-// Example:
+// For example, the following configuration configures the filter to authenticate the clients using
+// the API key from the header ``X-API-KEY``. And only the clients with the key ``fake-key`` are
```

Comment:
```suggestion
// the API key from the header ``X-API-KEY``. And only the clients with the key ``real-key`` are
```

The reasoning behind my suggestion is that fake-key doesn't sound like something that should be valid.